pulp_python/app/tasks/publish.py

```diff
@@ -104,7 +104,7 @@ def write_simple_api(publication):
         index.write(template.render(context))
 
     index_metadata = models.PublishedMetadata(
-        relative_path=simple_dir,
+        relative_path=simple_dir + "/",
```

Comment:
Why not set `simple_dir = 'simple/'` instead of adding the trailing slash twice?